crates/executor/src/executor.rs

```diff
@@ -149,7 +149,7 @@ pub fn commit_changes<DB: StateProvider>(
             db_account.account_state = AccountState::NotExisting;
             db_account.info = AccountInfo::default();
 
-            continue
+            continue;
```

Comment:
pls remove code chagnes

Reply:
Thanks for pointing this out! It looks like my "format on save" was enabled. I made sure to turn it off so this doesn't happen in future PRs. Sorry for the mixup! Everything should be fixed now and the only changes should be within `docs/repo/crates/executor.md`.